docs/workflow/gitlab.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "GitLab"
+toc: true
+weight: 592
+indent: true
+---
+
+# Continuous Delivery with GitLab
+
+## GitLab
+[GitLab][GitLab Kubernetes docs] supports deploying Kubernetes applications
+using GitLab pipelines today, but cloud service dependencies must be
+separately provisioned, connected, and secured. 
+
+## Provision cloud services for Kubernetes apps with Crossplane
+With GitLab 12.5, Crossplane is now available as a GitLab-managed app that can
```

Comment:
Should we provide context on what a GitLab-managed app is? Perhaps a link?

Reply:
good catch, link added